scripts/us_census/acs5yr/subject_tables/common/resolve_geo_id_test.py

```diff
@@ -13,8 +13,22 @@
 # limitations under the License.
 """Tests for resolve_geo_id."""
 
+import os
+import sys
 import unittest
-from .resolve_geo_id import convert_to_place_dcid
+
+# Allows the following module imports to work when running as a script
+# relative to data/scripts/
+sys.path.append(
+    os.path.sep.join([
+        '..' for x in filter(lambda x: x == os.path.sep,
+                             os.path.abspath(__file__).split('scripts/')[1])
+    ]))
+
+# module_dir_ is the path to where this test is running from.
+module_dir_ = os.path.dirname(__file__)
+
+from resolve_geo_id import convert_to_place_dcid
```

Comment:
Same as the previous comment, we may want to check if we need to replace `resolve_geo_id` in line 31 with `us_census.acs5yr.subject_tables.common.resolve_geo_id`, to resolve the ModuleNotFoundError in cloudbuild?

Reply:
reverted to earlier import.